src/Lint.jl

```diff
@@ -345,31 +345,36 @@ function lintdir{T<:AbstractString}(dir::T, ctx::LintContext=LintContext())
     ctx.messages
 end
 
+function readandwritethestream(conn)
+    println("Connection accepted")
+    # Get file, code length and code
+    file = strip(readline(conn))
+    println("file: ", file)
+    code_len = parse(Int, strip(readline(conn)))
+    println("Code bytes: ", code_len)
+    code = Compat.UTF8String(read(conn, code_len))
+    println("Code received")
+    # Do the linting
+    msgs = lintfile(file, code)
+    # Write response to socket
+    for i in msgs
+        write(conn, string(i))
+        write(conn, "\n")
+    end
+    # Blank line to indicate end of messages
+    write(conn, "\n")
+    println("Connection closed")
+    close(conn)
```

Comment:
it seems weird to me to close the connection in this function.  I think it's better to close it in a `finally` in the `lintserver` function.